contracts/test/L1ETHRegistry.t.sol

```diff
@@ -40,7 +41,7 @@ contract TestL1ETHRegistry is Test, ERC1155Holder, RegistryRolesMixin, EnhancedA
         datastore = new RegistryDatastore();
         ejectionController = new MockEjectionController();
         registryMetadata = new MockRegistryMetadata();
-        registry = new L1ETHRegistry(datastore, registryMetadata, address(ejectionController));
+        registry = new L1ETHRegistry(datastore, registryMetadata, IL1EjectionController(address(ejectionController)));
```

Comment:
i think you don't need this conversion, `MockEjectionController` is already implements IL1EjectionController, passing `ejectionController` should be fine.